content/automate/ManageIQ/System/CommonMethods/MiqAe.class/__methods__/weightedupdatestatus.rb

```diff
@@ -122,6 +136,11 @@ def main
                   # We set the task message.
                   if @handle.root['ae_state_step'] == 'on_error'
                     task.message = 'Failed'
+                    create_cleanup_request(task)
+                  elsif task.get_option('cancel_requested')
+                    task.message = 'Cancelled'
```

Comment:
Maybe `Canceled`? I am used to writing `Cancelled` too, but it seems in the US `Canceled` is more common.

Reply:
Yes. You're right about it. I've grep'd the code and everywhere it's `canceled`.